src/app/ui/timeline/timeline.cpp

```diff
@@ -1953,6 +1953,8 @@ void Timeline::onAfterLayerVisibilityChange(DocEvent& ev)
   layer_t layerIdx = getLayerIndex(ev.layer());
   if (layerIdx >= 0)
     invalidateRect(getPartBounds(Hit(PART_ROW_EYE_ICON, layerIdx)).offset(origin()));
+  if (docPref().onionskin.active())
+    docPref().onionskin.AfterChange();
```

Comment:
This works because the editor is connected to the `docPref().onionskin.AfterChange` signal and calls its `invalidate()` method. I believe that it would be better if we just call the editor's invalidate method directly as:

```suggestion
    m_editor->invalidate();
```

Reply:
The only problem using an invalidate for the current editor only is that it will not update other views/editors:

![image](https://github.com/user-attachments/assets/796a2376-f793-4404-943a-d63054983965)

We can do something like this:
```diff
    m_document->notifyGeneralUpdate();
```